package.json

```diff
@@ -2,15 +2,14 @@
   "private": true,
   "scripts": {
     "clean": "rimraf output && rimraf .pulp-cache",
-    "build": "jshint src && jscs src && pulp build --censor-lib --strict",
-    "test": "pulp build -I test"
```

Comment:
It was, but it's meant to be run by the user, since it enters a loop. I'm not sure what it was doing before.